packages/@n8n/nodes-langchain/nodes/memory/MemoryRedisChat/MemoryRedisChat.node.ts

```diff
@@ -143,12 +144,19 @@ export class MemoryRedisChat implements INodeType {
 		}
 		const redisChatHistory = new RedisChatMessageHistory(redisChatConfig);
 
-		const memory = new BufferMemory({
+		const memClass = this.getNode().typeVersion < 1.1 ? BufferMemory : BufferWindowMemory;
```

Comment:
Is it correct version here? The property is hidden for < 1.3

Reply:
ah, woops, that should indeed be 1.3; I'll fix that one